go/api/commands.go

```diff
@@ -150,8 +231,123 @@ type StringCommands interface {
 	//  The value of key after the decrement.
 	//
 	// For example:
-	//  result, err := client.Decr("key");
+	//  key: 1
+	//  result, err := client.DecrBy("key", 2);
+	//  result: -1
 	//
 	// [valkey.io]: https://valkey.io/commands/decrby/
 	DecrBy(key string, amount int64) (int64, error)
+
+	// Returns the length of the string value stored at key.
+	//
+	// See [valkey.io] for details.
+	//
+	// Parameters:
+	//  key - The key to check its length.
+	//
+	// Return value:
+	//  The length of the string value stored at key.
+	//  If key does not exist, it is treated as an empty string, and the command returns 0.
+	//
+	// For example:
+	//  key: value
+	//  result, err := client.Strlen("key");
+	//  result: 5
+	//
+	// [valkey.io]: https://valkey.io/commands/strlen/
+	Strlen(key string) (int64, error)
+
+	// Overwrites part of the string stored at key, starting at the specified offset, for the entire length of value.
+	// If the offset is larger than the current length of the string at key, the string is padded with zero bytes to make
+	// offset fit.
+	// Creates the key if it doesn't exist.
+	//
+	// See [valkey.io] for details.
+	//
+	// Parameters:
+	//  key    - The key of the string to update.
+	//  offset - The position in the string where value should be written.
+	//  value  - The string written with offset.
+	//
+	// Return value:
+	//  The length of the string stored at key after it was modified.
+	//
+	// For example:
```

Comment:
Please add example that include usual usage + usage with more than one byte chars. For example, if the value is Japanese char (I'm not sure about Go implementation, please validate me), it will be two bytes. 
So, changing just one byte will end up in garbage and change two bytes, or if you use an offset of three, you'll get just one padding between the char to the added value.

Reply:
Done.